src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -480,7 +486,7 @@ public void createSnapshot(Snapshot snapshot, List<String> rowIds) throws Interr
         "<selectStatements; separator=\" UNION ALL \">";
 
     private static final String validateSnapshotSizeTemplate =
-        "SELECT COUNT(1) FROM <snapshotProject>.<snapshot>.<dataRepoTable>";
+        "SELECT <rowId> FROM `<snapshotProject>.<snapshot>.<dataRepoTable>`";
```

Comment:
Doesn't this change the behavior of the query?

Reply:
It does bu in a way that makes it work :) If you look at line 560 where is use the output for validation it does:
```if (result.getTotalRows() <= 0) {...```
Which, if you leave it as a count, is always going to be 1.  So this mimics what we do elsewhere.  Oh wait, I'm going to limit to 1 so it doesn't return a bunch of rows potentially (since we're using this to check if this table is empty)